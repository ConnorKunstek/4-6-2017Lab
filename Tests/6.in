SDSSSSDDDDDSSSSSSSSSSSDSSSSSSS
DDDDDDDDDSSSSSSSSSSSSSSDSSSSSS
DSSSSSSSSSSSSSSSSSSSSSSSDSSSSS
DDDDDDDDDDDSSSSSSSSSSSSDSSSSSS
SSSSSSSSSSSSSSSSSSSSSSDSSSSSSS
SSSSSSSSSSSSSSSSSDDDDDSSSSSSSS
SSSSSSSSSSSSSSSSSFSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSDDDDDDSSSSSSSDDDDDSSSSSSSSS
SSSDDDDDDSSSSSSSDDDDDSSSSSSSSS
SSSDDDDDDSSSSSSSDDDDDSSSSSSSSS
SSSDDDDDDSSSSSSSSSSSSSSSSSSSSS
DDDDSSSSSSSSSSSSSSSSSSDDDDDDDD
SSSSSSSSSDSSSSSSSSSSSSDSSSSSSS
SSSSSSSSSDSSSSSSSSSSSSDSSSSSSS
SSSSSSSSSDDSSSSSSSSSSSDSSSSSSS
14 6
9 5
14 19
2 2
7 18
