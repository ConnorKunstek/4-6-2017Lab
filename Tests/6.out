Yes
No
No
No
Yes
Time: 0.005933 seconds

